tests/test_routes.py

```diff
@@ -76,7 +76,7 @@ def test_create_order_simple(self):
         new_order = resp.get_json()
         self.assertEqual(new_order["name"], order.name, "Names does not match")
         self.assertEqual(new_order["street"],
-                         order.street, "Address does not match")
+                         order.street, "Item does not match")
```

Comment:
Not sure what Item is doing here, this is a test for creating a simple order, not item.

Reply:
@bartconsedine